src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -33,7 +33,7 @@
 USE_CPP = _tkutl._read_env_var_cpp('TURI_AC_USE_CPP_PATH')
 
 def create(dataset, session_id, target, features=None, prediction_window=100,
-           validation_set='auto', max_iterations=10, batch_size=32, verbose=True):
+           validation_set='auto', max_iterations=10, batch_size=32, verbose=True, random_seed=None):
     """
```

Comment:
Please add a description for this new parameter in the Parameters section of the docstring.